Components/Services/OrderDataService.php

```diff
@@ -180,6 +181,8 @@ public function setOrderStatus($orderId, $newOrderStatusId)
      * @param string $transactionId
      *
      * @return OrderAndPaymentStatusResult|null
+     *
+     * @deprecated in 6.1.1, and will be removed with 7.0.0 without replacement
```

Comment:
```suggestion
     * @deprecated in 6.1.2, and will be removed with 7.0.0 without replacement
```

according to your changes in the plugin.xml :slightly_smiling_face: